Client/Clique/Modules/EventList/EventListTableViewCell.swift

```diff
@@ -20,7 +20,7 @@ class EventListTableViewCell: UITableViewCell {
     func configure(with event: Event) {
         _eventName.text = event.name
         _eventLocation.text = event.location
-        _eventTimestamp.text = event.timestamp
+        _eventTimestamp.text = event.timeStampToString(timestamp: event.timestamp)
```

Comment:
`timeStampToString` is not a good method name, because if I have `1234` as a number, will it produce `"1234"` as a String?

You should get a formatted date here, so please align with @Pa3kV about this, or add a `// TODO: <enter-message>` here so we know that this will change in the future.